apps/eth-wrapper/src/utils/Erc20Constants.ts

```diff
@@ -0,0 +1,17 @@
+export const Erc20 = [
+    "function approve(address _spender, uint256 _value) public returns (bool success)",
+    "function allowance(address _owner, address _spender) public view returns (uint256 remaining)",
+    "function balanceOf(address _owner) public view returns (uint256 balance)",
+    "event Approval(address indexed _owner, address indexed _spender, uint256 _value)"
+];
```

Comment:
Should be able to import the erc20 interface like this:

```
import IERC20 from "@openzeppelin/contracts/build/contracts/IERC20.json";
```

Reply:
I like Jose's way more because it won't include the JSON to the bundle and means less code shipped to the users. Maybe there are some optimisations for JSON so I suggest to test it